### ISSUE TITLE
Media type "application/octet-stream" is not supported

### ISSUE THREAD

**Issue by banex19** [GoogleCloudPlatform/healthcare-api-dicomweb-cli#45]
**Command:**
`dcmweb   https://staging-healthcare.sandbox.googleapis.com/v1/projects/dvet-test/locations/us-central1/datasets/1/dicomStores/1/dicomWeb retrieve --type="application/octet-stream" studies/1.2.840.113619.2.176.3596.3364818.7819.1259708454.105/series/1.2.840.113619.2.176.3596.3364818.7819.1259708454.108/instances/1.2.840.113619.2.176.3596.3364818.7271.1259708501.876`

**Output:**
```
2020-03-16 16:57:08,511 -- Saving files into ./
Traceback (most recent call last):
  File "/usr/local/google/home/dvet/.local/bin/dcmweb", line 10, in <module>
    sys.exit(main())
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/dcmweb/command_line.py", line 66, in main
    fire.Fire(host_wrapper)
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/fire/core.py", line 138, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/fire/core.py", line 471, in _Fire
    target=component.__name__)
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/fire/core.py", line 675, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/dcmweb/dcmweb.py", line 122, in retrieve
    self.requests.download_dicom_by_ids(ids, output, type)
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/dcmweb/requests_util.py", line 220, in download_dicom_by_ids
    return self.download_dicom(url, folder, file_name, mime_type)
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/dcmweb/requests_util.py", line 191, in download_dicom
    extension = extension_by_headers(content_type)
  File "/usr/local/google/home/dvet/.local/lib/python3.7/site-packages/dcmweb/requests_util.py", line 65, in extension_by_headers
    raise ValueError("unknown extension {}".format(content_type))
ValueError: unknown extension multipart/related; boundary=098f3ca1ff0cd52f4596b87d5cd2756b1ca69a18a1db1e6998a08691a667; transfer-syntax=*; type="application/octet-stream"
```

**Comment by pavertomato** [comment]
in this case frames returns with content-type=="application/octet-stream" should them be saved as files without extensions?

**Comment by dzelemba** [comment]
Maybe use a .raw extension?

**Comment by dzelemba** [comment]
Manually tested that this has been fixed in v1.0.1.
